### ISSUE TITLE
About FlowNet and optimizer

### ISSUE THREAD

**Issue by LanduRi** [feiyuhuahuo/Anomaly_Prediction#9]
Hello, I have seen that you have provided two kinds of flownet options. I would like to ask which performance is better. I tested it. It seems that Lite flownet and flownet2sd have similar performance. Is that right?

Another question, I read your commits history. You tried SGD optimizer, and then you changed back to Adam. Is Adam better in your experiment?

**Comment by feiyuhuahuo** [comment]
When I was visualizing these two flownets, their performance seems to be close. Both flownets were not trained on the dataset, and I don't know how to train them. So I used them directly. The paper said Lite flownet has better performance.
Adam can get better performance than SGD, but not too much, at most 2%, I can't remember clearly.

**Comment by LanduRi** [comment]
Thank you very much for your reply. 

Have you tried the segmented learning rate as the author did?
LRATE_G = [0.0002, 0.00002]
LRATE_G_BOUNDARIES = [50000]

**Comment by feiyuhuahuo** [comment]
I forgot it. But learning rate is not crucial if it is not changed too much.
